tests/playwright/sanity/modules/div-block/div-block.test.ts

```diff
@@ -122,4 +122,41 @@ test.describe( 'Div Block tests @div-block', () => {
 			await expect( secondContainerHandles ).toHaveScreenshot( 'normal-handles.png' );
 		} );
 	} );
+	test( 'Verify that text stroke style do not apply to empty view and frame handle elements', async ( { page, apiRequests }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+		const editor = await wpAdmin.openNewPage();
+		const divBlockId = await editor.addElement( { elType: 'e-div-block' }, 'document' );
+
+		const divBlock = editor.getPreviewFrame().locator( `[data-id="${ divBlockId }"]` );
+		await divBlock.waitFor();
+		const divBlockHandles = divBlock.locator( '.elementor-editor-element-settings' );
+		const divBlockEmptyView = divBlock.locator( '.elementor-empty-view' );
+
+		// Apply text stroke style to div block
+		await divBlock.evaluate( ( el ) => {
+			el.style.stroke = 'red';
+			el.style.strokeWidth = '10px';
+		} );
+
+		await divBlockEmptyView.waitFor();
+		const { stroke, strokeWidth } = await divBlockEmptyView.evaluate( ( el ) => {
+			return {
+				stroke: getComputedStyle( el ).stroke,
+				strokeWidth: getComputedStyle( el ).strokeWidth,
+			};
+		} );
```

Comment:
Looks great!
Can we extract this block to a function to avoid code duplication?

Reply:
I'm not sure we need to avoid it when it's in use twice in exactly the same test, or do you want to extract it as a general utility for getting styles from an element?
@davseve

Reply:
I extracted to a local function for now.